README.md

```diff
@@ -67,6 +67,7 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 - [Git and GitHub](#git-and-github)
 - [Good first issue finder](#good-first-issue-finder)
 - [Web3](#web3)
+- [Machine Learning](#machine-learning)
```

Comment:
Hey, please add `Machine learning` in alphabetical order

Reply:
Ok,
Thank you for the review